pkg/pillar/cmd/collectinfo/collectinfo.go

```diff
@@ -258,7 +258,7 @@ func (ci *collectInfo) subscribe(ps *pubsub.PubSub) {
 		AgentName:   "zedagent",
 		MyAgentName: agentName,
 		TopicImpl:   types.LOCConfig{},
-		Activate:    true,
+		Activate:    false,
```

Comment:
@christoph-zededa I assumed that you wanted to activate this when iterating over the map below, right?